libs/core/src/lib/dialog/dialog.component.ts

```diff
@@ -110,11 +112,16 @@ export class DialogComponent extends DialogBase implements OnInit, OnChanges, Af
     /** @hidden */
     private _onHidden: Subscription;
 
+    /** @hidden */
+    private _subscription = new Subscription();
```

Comment:
do not see that `_subscription` unsubscribes somehow.
btw we have `_subscriptions` in the base class you could use it here.

Reply:
> btw we have _subscriptions 
_subscriptions at the base changed from private to protected